spec/stacktrace_spec.rb

```diff
@@ -458,12 +452,6 @@ def out_project_trace(stacktrace)
       ]
     end
 
-    def out_project_trace(stacktrace)
-      stacktrace.map do |trace_line|
-        trace_line[:file] unless trace_line[:inProject]
-      end.compact
-    end
-
```

Comment:
This was refactored in the first commit, I figured I might as well avoid adding a third version of this method.